app/controllers/foreman_tasks/api/recurring_logics_controller.rb

```diff
@@ -39,9 +39,19 @@ def resource_class
         ForemanTasks::RecurringLogic
       end
 
+      api :POST, '/recurring_logics/bulk/destroy', N_('Delete recurring logics by search query')
+      param :search, String, :desc => N_('Search query'), :required => true
+      def bulk_destroy
+        scope = resource_scope.search_for(params[:search])
+        scope.each(&:destroy!)
+        render json: { destroyed: scope }
+        rescue ActiveRecord::RecordNotDestroyed => error
+          render json: { error: error, scope: scope }, status: :bad_request
```

Comment:
This should be indented one level to the left

Reply:
why? this line should happen only for the rescue

Reply:
If you do not use explicit `begin` and `end`, then `rescue` should be aligned to the same level as `def` and `end`

All these three are functionally equivalent, however the first one is preferred.

```ruby
def foo
  raise "Stop"
rescue => e
  puts e
end
```

```ruby
def foo
  raise "Stop"
  rescue => e
    puts e
end
```

```ruby
def foo
  begin
    raise "Stop"
  rescue => e
    puts e
  end
end
```

Reply:
Thanks for the explanation! fixed